conf_linux_public.py

```diff
@@ -141,6 +160,13 @@ def get_building_cmd(command, gcc_latest, enable_devtoolset):
 action('build',
        cmd=get_building_cmd(f'make -j{options["CPU_CORES"]}', GCC_LATEST, ENABLE_DEVTOOLSET))
 
+if args.get('fastboot'):
+    action('build',
+               cmd=('strip ./__bin/release/libmfxhw64-fastboot.so.{ENV[API_VERSION]}'))
```

Comment:
strip is a workaround. it seems that check for size should on next stage - install.